src/cosim/execution.cpp

```diff
@@ -151,8 +151,10 @@ class execution::impl
             do {
                 stepSize = step();
                 timer_.sleep(currentTime_);
-            } while (!stopped_ && !timed_out(endTime, currentTime_, stepSize));
-            return !stopped_;
+            }  while (!stopped_ && !timed_out(endTime, currentTime_, stepSize));
```

Comment:
The only change in this line seems to be an additional space which isn't supposed to be there.

Reply:
Space is removed in latest commit